workflow-init/profile/hawk.yml

```diff
@@ -236,3 +236,15 @@ component:
       - *backend_rest
       - *backend_grouping
       - *backend_search
+
+  configuration:
```

Comment:
each repo has a profile section with parameters that are used in standard workflows